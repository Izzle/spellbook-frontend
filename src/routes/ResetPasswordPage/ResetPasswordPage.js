import React, { Component } from 'react';

export default class ResetPasswordPage extends Component {
  render() {
    return (
      <section className='ResetPasswordPage'>
          <h2>Reset your account password here!</h2>
      </section>
    )
  }
}