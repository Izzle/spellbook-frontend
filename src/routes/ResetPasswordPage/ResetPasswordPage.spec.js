import React from 'react';
import renderer from 'react-test-renderer';
import ReactDOM from 'react-dom';
import ResetPasswordPage from './ResetPasswordPage';
import { BrowserRouter } from 'react-router-dom';


describe('<ResetPasswordPage />', () => {
    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<BrowserRouter><ResetPasswordPage /></BrowserRouter>, div);
        ReactDOM.unmountComponentAtNode(div);
    });
    it('renders this UI as expected', () => {
        const tree = renderer.create(<BrowserRouter><ResetPasswordPage /></BrowserRouter>).toJSON();
        expect(tree).toMatchSnapshot();
    });    
});