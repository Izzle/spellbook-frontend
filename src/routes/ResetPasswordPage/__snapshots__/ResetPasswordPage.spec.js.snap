// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ResetPasswordPage /> renders this UI as expected 1`] = `
<section
  className="ResetPasswordPage"
>
  <h2>
    Reset your account password here!
  </h2>
</section>
`;
